relay-sampling/src/lib.rs

```diff
@@ -54,13 +54,13 @@
 //! ## [`SamplingConfig`]
 //!
 //! ```json
-#![doc = include_str!("../tests/fixtures/sampling_config.json")]
+#![doc = include_str ! ("../tests/fixtures/sampling_config.json")]
 //! ```
 //!
 //! ## [`DynamicSamplingContext`]
 //!
 //! ```json
-#![doc = include_str!("../tests/fixtures/dynamic_sampling_context.json")]
+#![doc = include_str ! ("../tests/fixtures/dynamic_sampling_context.json")]
```

Comment:
Did the auto formatter make this change?

```suggestion
#![doc = include_str!("../tests/fixtures/dynamic_sampling_context.json")]
```